.coveragerc

```diff
@@ -0,0 +1,48 @@
+[run]
+# Enable branch coverage measurement
+branch = True
+
+# Source files to measure
+source = airbrakes
+
+# Patterns of files to omit from coverage
+omit =
+    */tests/*
+    */site-packages/*
+    setup.py
```

Comment:
we don't have a setup.py
```suggestion
```

Reply:
lol all of this was boiler plate leave me alone